src/utils/utils.js

```diff
@@ -131,7 +131,7 @@ export const rightDeepOnlyInlineInScopePath = createDOMPathGenerator(
     true,
 );
 
-export function findNode(domPath, findCallback = node => true, stopCallback = node => false) {
+export function findNode(domPath, findCallback = () => true, stopCallback = () => false) {
```

Comment:
If we remove this, it would be nice to have some jsdoc telling what param is passed to the callbacks.

Reply:
Personnellement je pense qu'il n'aurait pas fallu supprimer celui là. Moi j'avais suggéré de les laisser dans les définitions des fonctions "de base". C'était juste quand on overridait une fonction que je suggérais de les retirer.

Reply:
Et que fait-on des erreurs es-lint. Est-ce qu'on fait un commentaire de ligne `eslint-disable-next-line` ?

Reply:
Une autre option ce serait dans le code de faire un truc genre:
`if (!findCallback || findCallback(node))`

Reply:
Je pense que c'est vraiment important de limiter au max les eslint-disable, sinon c'est pas la peine d'utiliser un linter, mais si on décide de le faire, au minimum spécifier la rule, et idéalement ajouter un commentaire expliquant le pourquoi.